HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json

```diff
@@ -0,0 +1,506 @@
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorFaceToFacePrepopulation\".ConfirmingDiabetesCondition"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "5.2",
+          "type": "display",
+          "text": "Check Home Blood Glucose Lab form",
+          "enableWhen": [
+            {
+              "question": "5.1",
+              "operator": "=",
+              "answerBoolean": true
+            }
+          ]
+        },
+        {
+          "linkId": "5.3",
+          "type": "date",
+          "text": "Diagnosis confirmed at",
+          "enableWhen": [
+            {
+              "question": "5.1",
+              "operator": "=",
+              "answerBoolean": false
+            }
+          ],
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorFaceToFacePrepopulation\".DMConfirmedDate"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "6",
+      "text": "Chief complaint/history of present illness and associated signs/symptoms ",
+      "type": "text"
+    },
+    {
+      "linkId": "7",
+      "text": "Related past medical/surgical history",
+      "type": "open-choice",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+          "valueExpression": {
+            "language": "text/cql",
+            "expression": "\"BasicClinicalInfoPrepopulation\".AllProcedures"
+          }
+        }
+      ],
+      "repeats": "true",
+      "answerOption": []
+    },
+    {
+      "linkId": "8",
+      "text": "Medications",
+      "type": "open-choice",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+          "valueExpression": {
+            "language": "text/cql",
+            "expression": "\"BasicClinicalInfoPrepopulation\".MedicationList"
+          }
+        }
+      ],
+      "repeats": "true",
+      "answerOption": []
+    },
+    {
+      "linkId": "9",
+      "text": "Allergies",
+      "type": "open-choice",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+          "valueExpression": {
+            "language": "text/cql",
+            "expression": "\"BasicClinicalInfoPrepopulation\".Allergies"
+          }
+        }
+      ],
+      "repeats": "true",
+      "answerOption": []
+    },
+    {
+      "linkId": "10",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/review-of-system"
+        }
+      ],
+      "type": "display"
+    },
+    {
+      "linkId": "11",
```

Comment:
PhysicanExam is split into two sub-qs: VitalSigns and PE. You may need to wait for ROS branch to be merged